server/services/database/translations.js

```diff
@@ -0,0 +1,24 @@
+const { Pool } = require("pg");
+const config = require("../../config");
+const pool = new Pool(config);
+
+const createTranslation = ({ id, user_id, document_id, start_date }) => {
+  const sqlQuery = `INSERT INTO translations (id, user_id,document_id, start_date)
+  VALUES ($1,$2,$3,$4)`;
+  return pool
+    .query(sqlQuery, [id, user_id, document_id, start_date])
+    .then(result => result.rows)
+    .catch(error => console.error(error));
```

Comment:
You shouldn't catch the error and console log it here, otherwise it will silently fail if there is a problem. Maybe you can remove catch here, and catch the error in the route implementation, so if an error occurs, you can return a 400 status code with an error message.